lib/eslint.js

```diff
@@ -251,13 +265,13 @@ function run(console, runOptions) {
     }
   })
 
-  cli = new CLIEngine(options)
+  cli = new ESLint(options)
```

Comment:
this is part of one of eslint 8 changes, `CLIEngine` has been replace for `ESLint` as pointed on https://eslint.org/docs/user-guide/migrating-to-8.0.0#remove-cliengine